src/redux/account/account.type.ts

```
@@ -0,0 +1,19 @@
+export interface AccountState {
+  loading: {
+    getProfile: boolean;
+  };
+  user: Nullable<User>;
+  error: Nullable<string>;
+}
+
+/**
+ * Get Profile
+ */
+export interface GetProfileResponse extends BaseResponse<User> {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)